examples/hoomd-blue/Butane_ANN_OpenMM.md

```diff
@@ -0,0 +1,380 @@
+---
+jupyter:
```

Comment:
Link to the note-books from the README of the examples, to enable one-click starts.